docs/providers/thirdparty.md

```diff
@@ -85,6 +85,7 @@ Gateway | Composer Package | Maintainer
 [Vend](https://github.com/wheniwork/oauth2-vend) | wheniwork/oauth2-vend | [When I Work](https://github.com/wheniwork)
 [Vimeo](https://github.com/saf33r/oauth2-vimeo) | saf33r/oauth2-vimeo | [Safeer Ahmed](https://github.com/saf33r)
 [Vkontakte](https://github.com/j4k/oauth2-vkontakte) | j4k/oauth2-vkontakte | [Jack W](https://github.com/j4k)
+[Wordrpess](https://github.com/krombox/oauth2-wordpress) | krombox/oauth2-wordpress | [Roman Kapustian](https://github.com/krombox)
```

Comment:
Please fix the spelling for "WordPress."

Also, this won't prevent us from accepting this pull request, but I noticed that your library has a commented out `vardump()` in it, and also some tab characters mixed in with your spacing indentation. You might consider cleaning those up. :smile: